android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.kt

```diff
@@ -38,17 +39,22 @@ import androidx.core.net.toUri
 import androidx.core.os.bundleOf
 import androidx.fragment.app.replace
 import androidx.lifecycle.lifecycleScope
+import edu.berkeley.boinc.R.*
 import edu.berkeley.boinc.adapter.NavDrawerListAdapter
 import edu.berkeley.boinc.adapter.NavDrawerListAdapter.NavDrawerItem
 import edu.berkeley.boinc.attach.SelectionListActivity
 import edu.berkeley.boinc.client.ClientStatus
 import edu.berkeley.boinc.client.IMonitor
 import edu.berkeley.boinc.client.Monitor
 import edu.berkeley.boinc.databinding.MainBinding
+import edu.berkeley.boinc.rpcExtern.fragment.SettingsFragmentRpcExtern
 import edu.berkeley.boinc.ui.eventlog.EventLogActivity
 import edu.berkeley.boinc.utils.*
 import kotlinx.coroutines.launch
 
+
+// get Context using BOINCActivity.appContext
```

Comment:
The application context can be accessed from any `Context` object, not just from activities.

Reply:
Corrected